ceph-ansible-pull-requests/config/definitions/ceph-ansible-pull-requests.yml

```diff
@@ -0,0 +1,72 @@
+- scm:
+    name: ceph-ansible
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph-ansible.git
+          branches:
+            - ${sha1}
+          refspec: +refs/pull/*:refs/remotes/origin/pr/*
+          browser: auto
+          timeout: 20
+          skip-tag: true
+          wipe-workspace: false
+          basedir: "ceph-ansible"
+
+- scm:
+    name: ceph-build
```

Comment:
I was going to reply here, but since you are doing the magic hand there to like explain architectural features of this implementation, I rather leave here a GIF and explain in person
![1tflzmr](https://cloud.githubusercontent.com/assets/317847/12496480/005e7fac-c064-11e5-9858-2a93f4eb9118.gif)